$OpenBSD$

remove check for bundled sfml and its subdirectory
add include_directories
remove -ldl linker flag

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -6,14 +6,6 @@ option(CENDRIC_BUILD_SQLITE_SHELL "Build SQLite shell 
 option(CENDRIC_BUILD_DIALOGUE_TOOL "Build Dialogue Tool on Windows platform?" ON)
 option(CENDRIC_STEAM "Include steamworks API?" OFF)
 
-if (NOT IS_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/ext/sfml/src)
-    message(FATAL_ERROR
-        "Seems like some of the required dependencies are missing. "
-        "This can happen if you did not clone the project with the --recursive flag. "
-        "It is possible to recover by calling \"git submodule update --init --recursive\""
-    )
-endif()
-
 include(CheckCXXCompilerFlag)
 
 if (NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
@@ -26,8 +18,6 @@ if (APPLE)
 	option(SFML_BUILD_FRAMEWORKS "" ON)
 endif()
 
-add_subdirectory("${PROJECT_SOURCE_DIR}/ext/sfml")
-
 if (MSVC)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /MP")
     add_definitions (/D "_CRT_SECURE_NO_WARNINGS")
@@ -55,15 +45,18 @@ file(GLOB_RECURSE Cendric_FILES
 	"${PROJECT_SOURCE_DIR}/include/*.h"
 	"${PROJECT_SOURCE_DIR}/src/*.cpp"
 	"${PROJECT_SOURCE_DIR}/ext/Particles/*.cpp"
-	"${PROJECT_SOURCE_DIR}/ext/lua/src/*.c"
-	"${PROJECT_SOURCE_DIR}/ext/sqlite/sqlite3.c"
-	"${PROJECT_SOURCE_DIR}/ext/tinyxml2/*.cpp"
+	"${PROJECT_SOURCE_DIR}/ext/lua/src/*.c"
+	"${PROJECT_SOURCE_DIR}/ext/sqlite/sqlite3.c"
+	"${PROJECT_SOURCE_DIR}/ext/tinyxml2/*.cpp"
+
     if (CENDRIC_STEAM)
         "${PROJECT_SOURCE_DIR}/ext/steam-sdk/public/steam/*.h"
         "${PROJECT_SOURCE_DIR}/ext/steam-sdk/public/steam/*.cpp"
     endif()
 )
 
+include_directories(/usr/local/include)
+
 if (WIN32)
 	add_executable(Cendric
 		${Cendric_FILES}
@@ -102,7 +95,7 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
 	target_link_libraries(Cendric CendricApple)
 endif()
 
-set (CMAKE_CXX_LINK_EXECUTABLE "${CMAKE_CXX_LINK_EXECUTABLE} -lpthread -ldl")
+set (CMAKE_CXX_LINK_EXECUTABLE "${CMAKE_CXX_LINK_EXECUTABLE} -lpthread")
 
 target_link_libraries(Cendric sfml-system sfml-window sfml-graphics sfml-audio)
 
